src/gap.c

```diff
@@ -1523,6 +1523,23 @@ Obj FuncLOAD_DYN (
     if ( info == 0 )
         ErrorQuit( "call to init function failed", 0L, 0L );
 
+    /* info->type used to be a small integer -- test for this case         */
```

Comment:
How about using an inline `//` comment here? I go for those for all new comments, instead of trying to white-space align right sides of comments like this.